kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiSessionImpl.scala

```diff
@@ -96,6 +103,8 @@ class KyuubiSessionImpl(
     // we should call super.open before running launch engine operation
     super.open()
```

Comment:
nit: we can move `checkSessionAccessPathURIs`  before the `super.open()` to avoid creating operation log if conf is illegal.